src/ui.data-list/UDataList.vue

```diff
@@ -39,15 +39,24 @@
               />
             </slot>
 
-            <div v-bind="labelAttrs(element.isActive)">
-              <!--
-                @slot Use it to modify label.
-                @binding {object} item
-              -->
-              <slot name="label" :item="element">
-                {{ element[labelKey] }}
-              </slot>
-            </div>
+            <template v-if="element.isActive">
+              <div v-bind="labelCrossedAttrs">
```

Comment:
First the condition is wrong and not full.
Second - you may apply the condition in template to reduce template code.
Third: pass element.isActive as a slot prop `active`

`<div v-bind="element.isActive === undefined && element.isActive ? labelAttrs : labelCrossedAttrs"> ...`

`element.isActive === undefined && element.isActive` - move into helper function `isActive(element)`